mautic/mautic
# Add Auditlog tab to contact's detail page

| Q  | A
| --- | ---
| Bug fix? | N
| New feature? | Y
| Related user documentation PR URL | N/A
| Related developer documentation PR URL | N/A
| Issues addressed (#s or URLs) | N/A
| BC breaks? | N/A
| Deprecations? | N/A

#### Description:
This PR adds a new tab to the contact's details page that shows audit log entries for the contact. This will permit to debug changes to a contact (i.e. by integrations).

#### Steps to test this PR:
1. Apply this PR
2. Go to a contact's details page
3. You will see a new tab titled Audit log.
4. Click on it and it will show a table with 3 columns: User/Source Name (who/what updated the contact), Event Type and Timestamp.
